app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,10 @@
+# Dash-Board  Controller
+class DashboardController < ApplicationController
+  def allnotes
+    @created_notes = Note.where(user_id: current_user.id)
+    @contributor_notes = Role.where(
+      user_id: current_user.id,
+      role: 'Contributor')
```

Comment:
@sohilp393, Closing method call brace must be on the line after the last argument when opening brace is on a separate line from the first argument.
```
    @contributor_notes = Role.where(
      user_id: current_user.id,
      role: 'Contributor'
    )
```